modules/currency-l1/src/main/scala/org/tessellation/currency/l1/CurrencyL1App.scala

```diff
@@ -123,6 +130,20 @@ abstract class CurrencyL1App(
         .start(storages, services, healthChecks)
         .asResource
 
+      _ <- sdk.prioritySeedlist.flatMap(s => SortedSet.from(s).toNes) match {
```

Comment:
instead of match you can use `traverse` and/or `getOrElse`. also `asResource` can be executed once at the very end